scripts/checkMissingTranslations.ts

```diff
@@ -1,4 +1,4 @@
-import chalk from 'chalk'
+import * as chalk from 'chalk'
```

Comment:
Usually we don't need to use `* as`  syntax since we have `allowSyntheticDefaultImports: true`.
Is there any particular reason you changed this line?

Reply:
`npm run start` has  an error like below.

<img width="891" alt="Screen Shot 2020-06-15 at 12 01 08 PM" src="https://user-images.githubusercontent.com/8810755/84695827-4b83a180-af00-11ea-896d-db1d40594469.png">

Reply:
allow synthetics is not allowed in this `/scripts` ts config https://github.com/HospitalRun/hospitalrun-frontend/blob/master/scripts/tsconfig.json

Reply:
This is a bug on tsconfig.json side. I am going to fix this.

Reply:
I fixed the tsconfig.json on the master. Can you please merge master in kumikokashii:multiple-contact-info and try again leaving `import chalk from 'chalk'`

Reply:
✅